docs/docs/91-migrations.md

```diff
@@ -2,7 +2,11 @@
 
 Some versions need some changes to the server configuration or the pipeline configuration files.
 
-## 1.0.0 (next)
+## 1.1.0 (next)
```

Comment:
It's usually unclear what the next version will be - just omit this one?

Reply:
Changed to 'next' only